<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-88V4A8K" failures="3" tests="7" name="SeleniumProject" time="87.403" errors="0" timestamp="2024-04-09T07:50:55 IST">
  <testcase classname="TestNGConcepts.Login" name="registration" time="0.004">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNGConcepts.Login.registration(Login.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- registration -->
  <testcase classname="TestNGConcepts.Login" name="loginWithValidCredentials" time="5.333"/>
  <testcase classname="TestNGConcepts.Login" name="loginWithValidCredentials" time="4.86"/>
  <testcase classname="TestNGConcepts.Login" name="loginWithValidCredentials" time="4.724"/>
  <testcase classname="TestNGConcepts.Login" name="loginWithInValidEmailId" time="9.456">
    <failure type="java.lang.AssertionError" message="expected [My Account] but found [Account Login]">
      <![CDATA[java.lang.AssertionError: expected [My Account] but found [Account Login]
at day31.POM.MyAccount.validateLoginStatus(MyAccount.java:17)
at TestNGConcepts.Login.loginWithInValidEmailId(Login.java:72)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginWithInValidEmailId -->
  <testcase classname="TestNGConcepts.Login" name="loginWithInValidCredentials" time="0.0">
    <skipped/>
  </testcase> <!-- loginWithInValidCredentials -->
  <testcase classname="TestNGConcepts.Login" name="loginWithInValidPassword" time="4.469">
    <failure type="java.lang.AssertionError" message="expected [My Account] but found [Account Login]">
      <![CDATA[java.lang.AssertionError: expected [My Account] but found [Account Login]
at day31.POM.MyAccount.validateLoginStatus(MyAccount.java:17)
at TestNGConcepts.Login.loginWithInValidPassword(Login.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginWithInValidPassword -->
</testsuite> <!-- SeleniumProject -->
